libs/qCC_glWindow/src/ccGLWindow.cpp

```diff
@@ -1137,6 +1137,11 @@ bool ccGLWindow::event(QEvent* evt)
 	}
 	break;
 
+	case QEvent::Resize:
+	{
+		update();
+	}
+
```

Comment:
You would at least have to add a `return true;` statement. But mind that just after these new lines, you have a `#ifdef CC_GL_WINDOW_USE_QWINDOW` block, in which this event is already redefined. So you will need to move your own version in a #else block after the one below. i.e. line 1173.

Reply:
And I would be interested to know about your setup? I guess it's the non-stereo version? And with which version of Qt are you compiling?

Reply:
My Qt is 5.12.12.  
I tested both non-stereo and stereo mode, all has this issue.
It seems the event should pass to parent. Because after add `return true`, it looks more weird, can't show any elemnets.
Add I will move the code to `#else `block .

Reply:
1ef92605780f2c9d6d728a15e88a54fb58bc62e8